### PR TITLE
feat(bridge): Delete a service (#4380)

### LINKED ISSUE #4380: Delete a service in Bridge
## Use Story

As a user, I would like to delete a service using the Bridge; i.e., the counterpart to `keptn delete service --project=xyz`. 

#### Mockup

* Based on #4501, the settings page allows deleting a service:

image

* By going to the settings page and selecting a service, the user can delete it
* When selecting a service, a `Danger zone` section is provided that allows deleting the service
* A confirmation dialog asks the user to enter the name of the service
* After entering the name, the button (I understand ... ) is active
* If an error occurred, the error is shown in the confirmation dialog:

image

* After successfully deleting a service, the user stays at the settings page
* A notification informs the user about a successful deletion: 

image

* Finally, the next service in the list gets selected (in this case: carts-db)

### Acceptance criteria

- [x] Instead of using `keptn delete service` I can delete a service using the Bridge.
- [x] Proper error handling (e.g. when I cannot delete a svc) 

### Definition of Done
- [x]  Everything in acceptance criteria implemented and manually tested
- [x]  Added unit tests
- [x]  Put attention to test corner cases
- [x]  Added UI Tests for deleting a svc
- [ ]  Documentation updated

### PR DESCRIPTION
**Pull request by Kirdock**
## This PR
Adds the capability to delete a service

### Related Issues
Closes #4380 

### Notes
Branch based on #4500 

Services list:
image

Edit/Delete a Service
image

### CODE REVIEW

File: bridge/client/app/_components/ktb-service-settings-overview/ktb-service-settings-overview.component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
+
+  it('should show create button', () => {
+    // given
+    const creatButton = fixture.debugElement.query(By.css('button'));
```
Comment (laneli) [inline bridge/client/app/_components/ktb-service-settings-overview/ktb-service-settings-overview.component.spec.ts:39]: Typo: createButton
Also you extend the test for text "Create service" on the button

File: bridge/client/app/_views/ktb-settings-view/ktb-settings-view.component.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+::ng-deep .settings-section {
```
Comment (laneli) [inline bridge/client/app/_views/ktb-settings-view/ktb-settings-view.component.scss:1]: As it will probably be used in more than one component from now on, you could move it to `styles.scss` to omit the `::ng-deep` selector

File: bridge/client/app/_views/ktb-settings-view/ktb-settings-view.component.ts [hunk 3]
```diff
@@ -1,11 +1,10 @@
 import { Component } from '@angular/core';
-import { NotificationsService } from '../../_services/notifications.service';
 
 @Component({
   selector: 'ktb-settings-view',
   templateUrl: './ktb-settings-view.component.html',
-  styleUrls: [],
-  providers: [NotificationsService],
+  styleUrls: ['./ktb-settings-view.component.scss'],
```
Comment (laneli) [inline bridge/client/app/_views/ktb-settings-view/ktb-settings-view.component.ts:6]: If you move it to `styles.scss` don't forget to delete this and the file :)

**Review by laneli — CHANGES_REQUESTED** [review]

**Review by laneli — DISMISSED** [review]

File: bridge/client/app/_views/ktb-settings-view/ktb-settings-view.component.scss (hunk 2 above)
Comment (Kirdock) [reply bridge/client/app/_views/ktb-settings-view/ktb-settings-view.component.scss]: I would leave it here because this component is the parent for all settings

**Review by laneli — APPROVED** [review]
